src/index.ts

```diff
@@ -281,14 +324,14 @@ class FSMerger {
 
 export = FSMerger;
 namespace FSMerger {
-  export interface FS extends FSMerger {
-    existsSync: typeof existsSync,
-    lstatSync: typeof lstatSync,
-    statSync: typeof statSync,
-  }
+
+  export type FS =
+    Pick<typeof nodefs, 'readFileSync' | 'readdirSync' | 'readdir' | 'existsSync' | 'lstatSync' | 'statSync'>
+    & Pick<FSMerger, 'at' | 'readFileMeta' | 'entries' | 'relativePathTo'>;
```

Comment:
I think this is a much cleaner way to define the `FS` interface. Doing so didn't require the `FSMerger.FSMerger` interface so I just removed that.